### PR TITLE
Auto approve branch requests by package admins

### PR DESCRIPTION
**Pull request by tyll**

### CODE REVIEW

File: pkgdb2client/admin.py [hunk 1]
```diff
@@ -437,12 +465,18 @@ def do_process(args):
...
+                package_name = action['info']['pkg_name']
+                package = PKGDBCLIENT.get_package(
+                    package_name, branches=["master"]
+                )["packages"][0]
+
+                package = PKGDBCLIENT.get_package(
+                    package_name, branches=["master"],
                     namespace=action['info'].get('pkg_namespace', 'rpms'),
-                )
+                )["packages"][0]
```
Comment (pypingou) [inline pkgdb2client/admin.py]: I'm seeing two queries to pkgdb saved in the same variable here, left over code?

File: pkgdb2client/admin.py [hunk 2]
```diff
@@ -391,6 +390,35 @@ def __handle_request_branch(actionid, action):
...
+        # requested, it needs to be approved by a pkgdb admin.
+        if branch[0] == "f":
+            for acl in package["acls"]:
+                if acl["fas_name"] == action["user"] and \
+                        acl["acl"] == "approveacls" and \
+                        acl["status"] == "Approved":
+                    if len(msgs["bad"]) == 0:
+                        print("Auto approving Fedora Branch request for "
+                              "package admin {0}".format(action["user"]))
+                        return approve()
```
Comment (pypingou) [inline pkgdb2client/admin.py]: Just styling, but I think I would have approached it by setting ``decision`` to ``approve`` and call ``_ask_what_to_do`` only if it's not None and then left the rest of the logic un-changed (ie: not have the ``approve`` inner method, which I find a little harder to read).

File: pkgdb2client/admin.py (hunk 1 above)
Comment (tyll) [inline pkgdb2client/admin.py]: Thank you, yes, I guess this was a conflict merge artificact. I cleaned it.

File: pkgdb2client/admin.py (hunk 2 above)
Comment (tyll) [inline pkgdb2client/admin.py]: Ok, good idea, I adjusted this as well.

**Comment by pypingou** [comment]
Looks good to me, thanks!

**Pull request merged**